background/ncn_kdump.md

```diff
@@ -87,6 +87,13 @@ to be installed; the `crash` command can not thoroughly analyze a dump without t
 
     * Install from Artifactory if credentials are available.
 
+        1. This step should ONLY be performed if an http proxy is required to access a public endpoint on the internet for the purpose of downloading artifacts. CSM does NOT support the use of proxy servers for anything other than downloading artifacts from external endpoints. The http proxy variables must NOT be exported and must be unset after the desired artifacts are downloaded. Failure to unset the http proxy variables after downloading artifacts will cause many failures in subsuquent install steps.
```

Comment:
Doing a quick test on drax, it looks like we're going to need to export these variables in each of the cases.   So we'll need to remove the part about "must NOT be exported" but leave the "must be unset".

Reply:
Also, the "subsequent install steps" probably doesn't make sense in this case since this is more of an operational procedure that will likely be done at some point after the install.  So maybe something like "may cause failures in subsequent steps"